packages/frontend-2/lib/billing/composables/actions.ts

```diff
@@ -174,6 +203,18 @@ export const useBillingActions = () => {
           type: ToastNotificationType.Success,
           title: 'Your workspace plan was successfully updated'
         })
+
+        if (import.meta.server) {
+          await sendWebhook(defaultZapierWebhookUrl, {
+            workspaceId: workspace.id,
```

Comment:
Would be nice to also include Workspace name

Reply:
Good point, just completely forgot that one